Shops/ShopGoblinTinkerer.cs

```diff
@@ -45,87 +45,71 @@ public void InitShop(string currentShop)
                         shop.item[nextSlot++].SetDefaults(calamity.ItemType("Laudanum"));
                         shop.item[nextSlot++].SetDefaults(calamity.ItemType("StressPills"));*/
                     //}
-                    break;
+                    //break;
                 case "Miscellaneous":
                     if (NPC.downedPirates)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.GoldRing);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         shop.item[nextSlot].SetDefaults(ItemID.DiscountCard);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                         shop.item[nextSlot].SetDefaults(ItemID.LuckyCoin);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
                     shop.item[nextSlot].SetDefaults(ItemID.FlowerBoots);
-                    shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     if (Utils.multiKills(new short[] { NPCID.BlueJellyfish, NPCID.PinkJellyfish, NPCID.GreenJellyfish }) > 100)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.JellyfishNecklace);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
                     if (NPC.downedMechBoss1)
                     {
                         shop.item[nextSlot].SetDefaults(ItemID.NeptunesShell);
-                        shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost;
                     }
                     break;
                 case "Special":
-                    if (NPC.downedSlimeKing) { shop.item[nextSlot].SetDefaults(ItemID.RoyalGel); shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost; }
-                    if (NPC.downedBoss1) { shop.item[nextSlot].SetDefaults(ItemID.EoCShield); shop.item[nextSlot++].shopCustomPrice = MerchantsPlus.universalAccessoryCost; }
+                    if (NPC.downedSlimeKing) { shop.item[nextSlot].SetDefaults(ItemID.RoyalGel); }
+                    if (NPC.downedBoss1) { shop.item[nextSlot].SetDefaults(ItemID.EoCShield); }
```

Comment:
You need `++` otherwise everything will be merged into one slot.

Please review the prices.